.github/workflows/verify-build.yml

```diff
@@ -368,13 +368,18 @@ jobs:
       run: brew install autoconf automake libtool
       if: ${{ matrix.os == 'macos-latest' }}
 
-    - name: Setup coverage dependencies (only on linux)
+    - name: Setup dependencies (only on linux)
       run: |
         sudo apt-get install info install-info ;
         sudo pip install gcovr ;
         sudo apt-get install cppcheck ;
       if: ${{ matrix.os-type == 'ubuntu' }}
 
+    - name: Setup coverage dependencies (only on linux)
```

Comment:
I don't suspect this is going to do much as it is since gcovr is already installed in the previous step.

It might be worth removing gcovr from the previous step if we are anyway restricting it here (which is fine).

The other thing is the "success()" in the if statement. I think that will wait for the overall run to succeed before installing gcovr - I don't think that should be there.

Reply:
Done.

That said, I think this ran correctly even with the `&&success()`. (Does that check that all *prior* steps passed? If not, how would it know what steps are "the entire run"?)